### PR TITLE
cdk: add page

### PR DESCRIPTION
**Pull request by honarmand-io**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/common/cdk.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# aws cdk
+
+> CLI for AWS Cloud Development Kit (CDK).
```
Comment (navarroaxel) [inline pages/common/cdk.md:3]:
```suggestion
> A CLI for AWS Cloud Development Kit (CDK).
```

File: pages/common/cdk.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+- List the stacks in the app:
+
+`cdk ls`
+
+- Synthesize and print the CloudFormation template for the specified stack(s):
+
+`cdk synth {{stack_name}}`
+
+- Deploy the specified stack(s):
```
Comment (navarroaxel) [inline pages/common/cdk.md:14]: maybe
```suggestion
- Deploy a space-separated list of stacks:
```

File: pages/common/cdk.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+`cdk deploy {{stack_name}}`
+
+- Destroy the specified stack(s):
```
Comment (navarroaxel) [inline pages/common/cdk.md:18]:
```suggestion
- Destroy a space-separated list of stacks:
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]
Hey @honarmand-io thanks, I left some comments.

**Comment by ahonarmand** [comment]
Thanks for the review @navarroaxel !

File: pages/common/cdk.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
+# aws cdk
```
Comment (sbrl) [inline pages/common/cdk.md:1]: This needs to exactly match the name of the command being documented.

**Review by sbrl — CHANGES_REQUESTED** [review]

**Review by sbrl — APPROVED** [review]
Thanks, @honarmand-io!

**Pull request merged**